### PR TITLE
update tagline

### PR DESCRIPTION
**Pull request by ryanatkn**
the grammar reads strangely to me because of the "and" in the first of three points, maybe a tweak can fix it?

> Felt is a free and open source toolkit for hobbyists and human-scale communities, publishing to the web, and collaborating with information.

I think the grammar makes sense if we omit the commas, but it doesn't seem like what we meant:

> Felt is a free and open source toolkit for hobbyists and human-scale communities publishing to the web and collaborating with information.

It seems kind of wordy anyway. Maybe simplifying with phrasing we were going to use in a different context:

> Felt is a free and open source web toolkit for hobbyists and human-scale communities.

At the moment I think the last 2 points are implied and don't add much.

### CODE REVIEW

File: src/lib/project.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const project: Project = {
 		},
 		{
 			name: '@feltjs/felt-server',
-			summary: 'server for Felt, a toolkit for building and maintaining communities',
+			summary: 'server for Felt, a toolkit for hobbyists and human-scale communities',
```
Comment (greatbacon) [inline src/lib/project.ts:35]: I think for the server we had discussed describing it as `a programmable web platform for hobbyists & human-scale communities` could also throw in a "part of the Felt toolkit" or something.

**Comment by greatbacon** [comment]
> the grammar reads strangely to me because of the "and" in the first of three points, maybe a tweak can fix it?
> 
> > Felt is a free and open source toolkit for hobbyists and human-scale communities, publishing to the web, and collaborating with information.
> 
> I think the grammar makes sense if we omit the commas, but it doesn't seem like what we meant:
> 
> > Felt is a free and open source toolkit for hobbyists and human-scale communities publishing to the web and collaborating with information.
> 
> It seems kind of wordy anyway. Maybe simplifying with phrasing we were going to use in a different context:
> 
> > Felt is a free and open source web toolkit for hobbyists and human-scale communities.
> 
> At the moment I think the last 2 points are implied and don't add much.

I think the grammatical purpose of the `for` in the tagline has shifted from "what it does" to "who it is for". 

We could keep the functional elements though by going 

```
Felt is a free and open source toolkit for hobbyists and human-scale communities. 
It facilitates building and maintaining those communities, publishing to the web, 
and collaborating with information.
```

Although to your last point, I'm also fine with just trimming things down too.

File: src/lib/project.ts (hunk 1 above)
Comment (ryanatkn) [reply src/lib/project.ts:35]: whoops I was blindly find/replacing

**Comment by ryanatkn** [comment]
Ok cool fixed the felt-server oversight, I think it looks good. And if we want to add more info to the opening description beyond the one line - but the old text does feel redundant and techy.

File: src/routes/about/+page.svelte [hunk 2]
```diff
@@ -12,13 +12,11 @@
 <div class="width_md prose">
 	<section class="panel prose">
 		<p>
-			Felt's mission is to make our lives better with technology. Our main product is <a
+			Felt's mission to make our lives better with technology. Our main product is <a
```
Comment (greatbacon) [inline src/routes/about/+page.svelte:15]:
```suggestion
			Felt's mission is to make our lives better with technology. Our main product is <a
```
I think we still need the `is` here

**Review by greatbacon — APPROVED** [review]
Had one last nit, otherwise looks good!

**Pull request merged**